builder/dispatchers.go

```diff
@@ -261,7 +266,11 @@ func run(b *Builder, args []string, attributes map[string]bool, original string)
 	args = handleJsonArgs(args, attributes)
 
 	if !attributes["json"] {
-		args = append([]string{"/bin/sh", "-c"}, args...)
+		if runtime.GOOS != "windows" {
+			args = append([]string{"/bin/sh", "-c"}, args...)
+		} else {
+			args = append([]string{"cmd", "/S /C "}, args...)
```

Comment:
I just testings on my windows box and you're right:
`cmd "/S /C" ...`
does seem ok.  But I'd prefer if you removed the space afterthe "/C" because leaving it there might lead some people to think we're concatenating those strings (hence need the space) but we're not.